_data/sites.json

```
@@ -5752,6 +5783,17 @@
         ]
     },
 
+    {
+		    "name": "HitFilm",
+		    "url": "https://fxhome.com/privacy",
+		    "difficulty": "hard",
+		    "notes": "Must email to privacy@fxhome.com in order to get personal data deleted.",
```

```suggestion
		    "notes": "Must email them in order to get personal data deleted.",
```
Let's omit the e-mail from here, since it's already in its own field below.